integration/tests/build.gradle

```diff
@@ -18,7 +18,6 @@ android {
     // Set the target app project. The module specified here should contain the production code
     // test should run against.
     targetProjectPath ":app"
-    targetVariant "debug"
```

Comment:
Removing because `android.targetVariant is deprecated, all variants are now tested.`.